src/main/java/com/ashkan/moviesapi/constants/constants.java

```diff
@@ -0,0 +1,16 @@
+package com.ashkan.moviesapi.constants;
+
+import java.util.Arrays;
+import java.util.List;
+
+public final class constants {
+    private static final Long DAYS = (long) 3;
+    public static final Long DAYS_TO_ADD = DAYS * 24 * 60 * 60 * 1000;
+    public static final List<String> Rates = Arrays.asList(
```

Comment:
Maybe you can use for this an Enum class.  Take a look if that fits with what you need.

Reply:
List was used because i could avoid an iterator or switch using Enum.So for example using list will be :
`if(Constants.RATES.contains(movie.getRate()))`{
    //POST
}else{
    //404
}

Reply:
But if I have a Table in the database with a Rate list it will be better use Enum because i will be using it as a" Map" or "Dictionary" inserting ids equivalent instead of Strings,so it will be better for Data Base storage